dev/config/MASTER.yaml

```diff
@@ -20,42 +20,54 @@ mesos_options:
   dockercfg_location: file:///root/.dockercfg
 
 jobs:
-    - name: testjob0
```

Comment:
instead of tracking changes to this file, i think we should rename it to MASTER.yaml.example and gitignore MASTER.yaml

Reply:
I can do that!

Reply:
Actually.. idk, I think we should just not care. I'll remove my comments, but I would like to make `make dev` always work out of the box with a sane config